scripts/lint

```diff
@@ -23,6 +23,13 @@ if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
         yapf -dpr ${DIRS_TO_CHECK[@]}
         # Lint
         flake8 ${DIRS_TO_CHECK[@]}
+        # Type checking
+        for module in src/*/; do
+            if [[ "$module" == *.egg-info/ ]] ; then continue; fi
+            for submodule in "$module"/*; do
+                mypy -p "$(basename "$module").$(basename "$submodule")"
```

Comment:
I've found in the past that errors can slip through if `--namespace-packages` isn't included.

Reply:
Yeah, it's in `mypy.ini` (which we need for the module ignore list). `mypy` automatically picks it up by location. There would be a ton of "missing module" errors otherwise.